clients/ui/bff/internal/mocks/static_data_mock.go

```diff
@@ -828,9 +828,10 @@ func GetCatalogSourceMocks() []models.CatalogSource {
 			Labels:  []string{"Sample category 1"},
 		},
 		{
-			Id:     "dora",
-			Name:   "Dora source",
-			Labels: []string{},
+			Id:      "dora",
+			Name:    "Dora source",
+			Enabled: &disabled,
+			Labels:  []string{},
```

Comment:
@YuliaKrimerman would you like to revert this changes in this PR to make `community and custom` visible in mock mode. Because this changes were mentioned in PR description to test this PR. so should we revert it or is this changes intentional?